sqs4.tf

```
@@ -0,0 +1,18 @@
+locals {
+  sqs4_name = length(var.sqs4_name_override) > 0 ? var.sqs4_name_override : join("-", [var.name, "1"])
```

Shouldn't it be like so (`4` at the very end)?
```suggestion
  sqs4_name = length(var.sqs4_name_override) > 0 ? var.sqs4_name_override : join("-", [var.name, "4"])
```